src/citrine/informatics/predictors/graph_predictor.py

```diff
@@ -35,11 +35,9 @@ class GraphPredictor(Serializable['GraphPredictor'], Predictor):
 
     predictors = _properties.List(_properties.Union(
         [_properties.UUID, _properties.Object(Predictor)]), 'config.predictors')
-    training_data = _properties.List(_properties.Object(DataSource), 'config.training_data')
+    training_data = _properties.List(
+        _properties.Object(DataSource), 'config.training_data', default=[])
     typ = _properties.String('config.type', default='Graph', deserializable=False)
-    # Graph predictors may not be embedded in other predictors, hence while status is optional
-    # for deserializing most predictors, it is required for deserializing a graph
-    status = _properties.String('status', serializable=False)
```

Comment:
When returned via check-for-updates, the predictor isn't actually registered so it has no status.  Removing this line causes the `status` definition in `Predictor` to kick in, which lists this as optional.